nvidia-nano/setup-guides/configure_ansible.md

```diff
@@ -39,22 +41,23 @@ This guide shows how to access `root` on the nano from another computer
           inet 000.000.0.000
           ```
           >Note: Write down your Jetson Nano's ip address it is very important in the following steps
-  2. Run command `sudo su`
-      1. enter in your password ie `waggle`
-
-  3. Set root user password
-      1. run command `passwd root`
-
-      2. set password to `waggle`
-
-  4. Enable root user ssh login
-      1. run command `vim /etc/ssh/sshd_config`
-          1. Go to the bottom of the file
-          2. Enter in `PermitRootLogin yes` and `PasswordAuthentication yes` (Make sure they are in seperate lines)
-          3. Save and exit the file
-          > Note: In vim to enter in insert mode press 'i'. To exit insert mode press 'esc'. To save and quit type in ':wq' and press enter when not in insert mode
-      2. run command `service sshd restart` to restart ssh service
-
-> Note: The jetson nano can NOT have an IP on the ethernet in the 10.42.0.0 IP space as k3s internal network uses that subnet.
-
-Continue to [Setting up Ansible on your computer](./configuring_ansible.md)
\ No newline at end of file
+  3. Open a terminal on your computer and travel to your home directory (`cd ~` on macOS)
+  4. Clone this repository using this command:
+        ```
+        git clone https://github.com/waggle-sensor/node-platforms.git
+        ```
+  5. Travel into the folder that is holding the Ansible Playbooks using this command:
+        ```
+        cd ~/node-platforms/nvidia-nano
+        ```
+        >Note: Everytime you run an ansible commmand make sure you are in this directory
+  6. Replace `{ip}` in the `ansible_inventory` file with your Jetson Nano's ip address using vim or any other text editor program
```

Comment:
Is this still true?  I think u removed the `{ip}` part right?

Reply:
Yeah, I accidentally removed it from the inventory file. `{ip}` should replaced `0.0.0.0`.

Reply:
will fix in a future PR